openassetio_mediacreation/README.md

```diff
@@ -0,0 +1,11 @@
+# OpenAssetIO-MediaCreation-Package
+
+Openassetio-mediacreation is automatically generated using
```

Comment:
[consider] Odd capitalisation

Reply:
I would have thought that the start of a sentence gets a capital even if the word happens to be a package name, but don't mind either way. [414e9b0](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/414e9b0421b0f25396342ef16c6fa63a7772df28)

Reply:
I think if you're referring to the project, then it's "OpenAssetIO-MediaCreation", if you're referring to the package, then it's "The `openassetio-mediacreation` package".

Reply:
I think that's been my rule-of-thumb.

Reply:
Oh, blimey, Okay I see what you're getting at. [4413021](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/44130219b8781bd15148affc5a775c1ffc016501)

Reply:
The merge button is just too darn attractive, so i'm hitting it.